pkg/processor/build/runtime/python/runtime.go

```diff
@@ -43,6 +43,7 @@ func (p *python) GetProcessorDockerfileInfo(onbuildImageRegistry string) (*runti
 	}
 
 	if p.FunctionConfig.Spec.Runtime == "python:2.7" {
+		p.Logger.Warn("Nuclio will drop support for Python 2.7 within the next minor version")
```

Comment:
```suggestion
		p.Logger.Warn("Python 2.7 runtime is deprecated. Nuclio will drop support for Python 2.7 runtime as of version 1.6.0. Please migrate your code to Python 3.6")
```
No one remembers which versions is the next minor one since they probably don't remember which version they are using now :)